src/js/elements/mapchip/legend.js

```diff
@@ -0,0 +1,51 @@
+import {format as d3format} from 'd3-format';
+
+import { Observable } from "../../utils";
+
+const lightStart = 3;
+
+export class Legend extends Observable {
```

Comment:
you should be extending Component, not observable

Reply:
done